LibreNMS/Util/Url.php

```diff
@@ -225,13 +225,7 @@ public static function sensorLink($sensor, $text = null, $type = null, $overlib
      */
     public static function deviceUrl($device, $vars = [])
     {
-        $routeParams = [is_numeric($device) ? $device : $device->device_id];
```

Comment:
Is this change really working? its used for the device page tabs

Reply:
It generates the right Link to the device. But you are right, I haven't tested the tab vars...

Reply:
Seems to work.

https://exaple.com/librenms/ports/errors=yes

![Screenshot_2021-05-26_20-29-27](https://user-images.githubusercontent.com/11366337/119713107-a6887d00-be61-11eb-96f8-86d5cfbe35a1.png)

Reply:
Here is the problem @mpikzink 
That is the legacy format for device urls... I'm trying to migrate them to the new format.  This basically undoes that and routes all links through the compatibility routes to the device pages.

Reply:
@murrant Of course, that's not a good idea on my part then. I don't understand why only this page doesn't work with you, the other pages that have already been converted to Blade work without problems. Currently I have no idea how I can fix the problem differently do you have any idea how I can get further.